MetaMask/metamask-extension
# Creating stories for `LoadingHeartBeat` and `LoadingIndicator`

## Explanation

- Created stories for `loading-heartbeat` and `loading-indicator`
- `ui/components/ui/loading-heartbeat`
- `ui/components/ui/loading-indicator`


## Screenshots/Screencaps



### Before


![image](https://github.com/MetaMask/metamask-extension/assets/79097544/b8f74526-86b1-4d5e-8929-3ae253013e88)
![image](https://github.com/MetaMask/metamask-extension/assets/79097544/f4d2b884-6bb7-45f0-98e5-a3ea368387c8)

### After


### LoadingHeartbeat
![image](https://github.com/MetaMask/metamask-extension/assets/79097544/a1ce23e4-2e4c-493c-a958-65df3bf3959f)
![image](https://github.com/MetaMask/metamask-extension/assets/79097544/a715f4b1-1410-4ba9-8c35-339a85ace284)

### LoadingIndicator
When `isLoading` is set to `false`
![image](https://github.com/MetaMask/metamask-extension/assets/79097544/e8cf364f-2ead-4a7c-9681-de2b4458ed6e)

When `isLoading` is set to `true`
![image](https://github.com/MetaMask/metamask-extension/assets/79097544/e8377907-fa56-47ac-9953-5a5404039227)
![image](https://github.com/MetaMask/metamask-extension/assets/79097544/02802d53-b3ad-4c92-a7bb-4bf00cea278c)


## Manual Testing Steps

- Pull this branch
- Run `yarn storybook`
- check `LoadingHeartBeat` and `LoadingIndicator` stories



## Pre-merge author checklist


## Pre-merge reviewer checklist


If further QA is required (e.g. new feature, complex testing steps, large refactor), add the `Extension QA Board` label.

In this case, a QA Engineer approval will be be required.
